source/deployment/cluster.rst

```diff
@@ -265,6 +264,20 @@ In Mattermost v4.2 and later, a cluster leader election process assigns any sche
 
 The process is based on a widely used `bully leader election algorithm <https://en.wikipedia.org/wiki/Bully_algorithm>`_ where the process with the lowest node ID number from amongst the non-failed processes is selected as the "leader". 
 
+Job Server
+^^^^^^^^^^^^^^^^
+
+Mattermost schedules and completes periodic tasks via the `job server <https://docs.mattermost.com/administration/config-settings.html#jobs>`_. These tasks include:
```

Comment:
In the setup you describe below, it isn't *scheduling* periodic tasks via the jobserver, just running them on the jobserver.